src/plugins/home/server/services/tutorials/tutorials_registry.ts

```diff
@@ -14,12 +14,50 @@ import {
 } from './lib/tutorials_registry_types';
 import { tutorialSchema } from './lib/tutorial_schema';
 import { builtInTutorials } from '../../tutorials/register';
+import { CustomIntegrationsPluginSetup } from '../../../../custom_integrations/server';
+import { Category, CATEGORY_DISPLAY } from '../../../../custom_integrations/common';
+import { HOME_APP_BASE_PATH } from '../../../common/constants';
+
+const emptyContext = {};
+
+function registerTutorialWithCustomIntegrations(
+  customIntegrations: CustomIntegrationsPluginSetup,
+  provider: TutorialProvider
+) {
+  const tutorial = provider(emptyContext);
```

Comment:
We're already invoking the spec provider in `registerTutorial`, I don't think it's necessary to do it a second time here, it is? Can't we just change L82

```ts
tutorialSchema.validate(specProvider(emptyContext));
```

to 

```ts
const tutorial = specProvider(emptyContext)
tutorialSchema.validate(tutorial);
```

and then pass the resulting `TutorialSchema` down to `registerTutorialWithCustomIntegrations` instead of passing the `TutorialProvider`?